examples/member_iam/variables.tf

```diff
@@ -18,3 +18,7 @@ variable "project_id" {
   description = "Project id"
   type        = string
 }
+variable "prefix" {
```

Comment:
No need to add this as a variable on the example, we can hardcode to `serviceAccount`.

Reply:
you have right ;)